UnityProject/Assets/Scripts/Electricity/PoweredDevices/APC.cs

```diff
@@ -138,10 +99,40 @@ public void PowerNetworkUpdate()
 				}
 			}
 		}
-
+		BatteryCharging = false;
+		foreach (var bat in ConnectedDepartmentBatteries) {
+			if (bat.BatterySupplyingModule.ChargingWatts > 0) {
+				BatteryCharging = true;
+			}
+		}
 		Voltage = ElectricalNodeControl.Node.Data.ActualVoltage;
 		Current = ElectricalNodeControl.Node.Data.CurrentInWire;
 		HandleDevices();
+		UpdateDisplay();
+	}
+
+	public void UpdateDisplay() { 		
+		if (Voltage > 270)
+		{
+			State = APCState.Critical;
+		}
+		else if (Voltage > 219f)
+		{
+			State = APCState.Full;
+		}
+		else if (Voltage > 0f)
```

Comment:
Is there no mistake?
> 270 = critical
> 219 = full
> 0 = critical
else critical?

Reply:
it makes sense if  you count the last one as off,  but I changed it so it goes to critical instead,  so you can see that there is no power meant for the tiny little cell inside that keeps the APC  powered